src/app/[locale]/viewer/_components/ManualAasViewerInput.tsx

```diff
@@ -45,20 +45,28 @@ export function ManualAASViewerInput(props: { focus: boolean }) {
         try {
             setIsLoading(true);
 
-            const { registrySearchResult, aasId} = await getAasFromExternalServices(val);
-            const aas =
-                registrySearchResult != null
-                    ? registrySearchResult.registryAas
-                    : await repositoryClient.getAssetAdministrationShellById(encodeBase64(aasId));
-            
-            setAas(aas);
-            registrySearchResult?.registryAasData != null
-                ? setRegistryAasData({
-                    submodelDescriptors: registrySearchResult?.registryAasData?.submodelDescriptors,
-                    aasRegistryRepositoryOrigin: registrySearchResult?.registryAasData?.aasRegistryRepositoryOrigin })
-                : setRegistryAasData(null);
+            const aasIds = await handleAasDiscoverySearch(val);
+            if (aasIds && aasIds.length > 1) {
+                navigate.push(`/viewer/discovery?aasId=${val}`);
+            } else {
+                const aasId = aasIds && aasIds.length === 1 ? aasIds[0] : val;
```

Comment:
I believe this is here if `val` is not an asset Id but an AAS ID?
Can you explain that in a comment above somewhere?